PWM_NativePin.h

```diff
@@ -17,7 +18,8 @@ class PWM_NativePin : public PWM_Pins {
 
     // Give it a pin and int and you get PWM
     void set(int pin, int brightness) { analogWrite(pin, brightness); }
-    void set(int pin, float brightness) { set(pin, (int) brightness); } // tolerate float
+    // A float is 0.0 ... 1.0, which will be mapped to the RANGE
+    void set(int pin, float brightness) { set(pin, (int) brightness * RANGE); }
```

Comment:
`set( pin, 0.9)` (a float) now assumes you are using 0..1 range, and adjusts to whatever our RANGE is.